modules/test/playwright/pages/users-admin-web/EditOrganizationPage.ts

```diff
@@ -48,4 +60,16 @@ export class EditOrganizationPage {
 			trigger: this.editOrgLaborIconMenu,
 		});
 	}
+
+	async createOrganizationSite(organizationName: string) {
+		await (
+			await this.usersAndOrganizationsPage.organizationActionsMenu(
+				organizationName
+			)
+		).click();
+		await this.organizationEditMenuItem.click();
+		await this.organizationSiteLink.click();
+		await this.createSiteToggle.check();
+		await this.organizationSiteSaveButton.click();
```

Comment:
can you please remove this and move the instruction directly in the test? I think we will not using this so much in the future